### PR TITLE
meta descriptions for service mesh patterns and layer5&hashicoop

### PR DESCRIPTION
**Pull request by AbiodunVlad**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit 7e3d9222b59b565f0ddfa5e1c00347c9a804c862 at: https://651d6f3d65125073293d787c--layer5.netlify.app

**Comment by AbiodunVlad** [comment]
Meta descriptions.

**Comment by AbiodunVlad** [comment]
> hey @AbiodunVlad could you please edit the description with the issue it is fixing? Like This PR fixes #issue_no.

Done. Thank you for the lead.

**Comment by vishalvivekm** [comment]
@AbiodunVlad 
Let's discuss this on website call today at 5:30 PM IST / 7 AM Central time. Pleasde add this as an agenda item in the [meeting minutes](https://docs.google.com/document/d/1XczAHXVe2FIWPqiF57ospJ43zw5cZQ7ui8mn39v5EvA/edit#heading=h.lohhtewfwima)

**Review by Mohith234 — COMMENTED** [review]
@AbiodunVlad The changes made don't seem to update the meta description of those webpages when visited in the preview. Do check and fix it.

**Comment by Mohith234** [comment]
@AbiodunVlad If you look into the `templates/news-single.js` the graphql query doesn't seem to extract the description from the `.mdx` files. Add it there and also pass it into the SEO component as a prop.

For reference check this [PR](https://github.com/layer5io/layer5/pull/4898)

**Comment by l5io** [comment]
🚀 Preview for commit 1119f49f75b936f485bbf05bf957470f8eaa7fbe at: https://6526ab53c0461f6ab52d093a--layer5.netlify.app

File: src/collections/service-mesh-books/istio-up-and-running/index.mdx [hunk 1]
```diff
@@ -5,19 +5,30 @@ cover: ./istio-up-and-running.webp
 thumbnail: ./istio-up-and-running_top.png
 author: "Lee Calcote and Zack Butcher"
 abstract: "A must-read for anyone looking to get started with Istio."
+description: "Layer5 and HashiCorp join forces to launch a groundbreaking partnership, bringing advanced cloud-native infrastructure solutions for containerized and non-containerized workloads through Meshery and Consul integration. Explore the power of Meshery's support for Consul service mesh, including sample applications, performance management, and experimental support for WebAssembly, demonstrated at DockerCon 2020 for near-native fine-grained traffic control."
```
Comment (leecalcote) [inline src/collections/service-mesh-books/istio-up-and-running/index.mdx:8]: This description is for the Layer5 and HashiCorp partnership, not for the Istio: Up and Running book.

**Review by leecalcote — CHANGES_REQUESTED** [review]

**Comment by l5io** [comment]
🚀 Preview for commit ed1e9147a035116fe0b8da4e927ca26701bbf657 at: https://652d5a72ef45b12bfc04c48e--layer5.netlify.app

File: src/collections/service-mesh-books/istio-up-and-running/index.mdx (hunk 1 above)
Comment (AbiodunVlad) [reply src/collections/service-mesh-books/istio-up-and-running/index.mdx]: Hi @Mohith234 I have fixed the problem @leecalcote pointed. Kindly give it a final look. Thank you.

File: src/collections/service-mesh-books/istio-up-and-running/index.mdx [hunk 2]
```diff
@@ -5,19 +5,30 @@ cover: ./istio-up-and-running.webp
 thumbnail: ./istio-up-and-running_top.png
 author: "Lee Calcote and Zack Butcher"
 abstract: "A must-read for anyone looking to get started with Istio."
+
```
Comment (leecalcote) [inline src/collections/service-mesh-books/istio-up-and-running/index.mdx:8]: @AbiodunVlad is there a description for this page / book?

**Comment by l5io** [comment]
🚀 Preview for commit 9363ce8f42190ecd842b7c2d124c00d246a1222b at: https://653685f4cdbf46267c7c8dcd--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit e9eb64be920ecd14ddffa85f0a80bd61010ba817 at: https://653a77d6d509bf083f41bf2e--layer5.netlify.app

**Comment by l5io** [comment]
🚀 Preview for commit 35b52bac5c2abc49c6f62a340da51c0872287fef at: https://653a7f1ed2ee0103c5b16aaa--layer5.netlify.app

**Comment by Mohith234** [comment]
@vishalvivekm Good to merge ...

**Comment by Mohith234** [comment]
Thanks for contributing @AbiodunVlad 🚀

**Pull request merged**